File: Python/1.py
def sum_pares(lista_pares):
    lista_pares = lista_pares.split(',')
    lista_resultado = [];
    for i in lista_pares:
        par = i.split(' ')
        lista_resultado.append(int(par[0]) + int(par[1]))       
    return lista_resultado

print(sum_pares("1 2,3 4,5 6,7 8"))

File: Scala/1.scala
object ListaPares {
  def main(args: Array[String]): Unit = {
    print("\n______________Lista de Pares________________\n")
    print("Introduce la lista en formato: '1 2,3 4,5 6,7 8'\n> ")
    val cad : String = scala.io.StdIn.readLine
    val alist : Array[String] = cad.split(',')
    var res : Array[Int] = Array()
    var a : Int = 0
    var b : Int = 0
    for ( m1 <-alist )
        {
            var num = m1.split(' ')
            a = num(0).toInt
            b = num(1).toInt
            res = res :+ (a+b)
        }
    print("[ ")
    for ( x <- res ) 
    {
      print(x+" ")
    }
    print("]")

  }
}